src/app/grid/quickview.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import {QuickviewComponent} from "./quickview.component";
+
+describe('QuickviewComponent', () => {
```

Comment:
Ideally here you would test that the component contains the correct information. There should also be a test that verifies the correct UI behaviour, i.e. when the user clicks the detail page shows.